src/components/SmallCallout/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+export function SmallCallout({ children, serverity = "warning" }: { children: React.ReactNode; serverity: string }) {
```

Comment:
```suggestion
export function SmallCallout({ children, severity = "warning" }: { children: React.ReactNode; severity?: string }) {
```

- Spell check `serverity -> severity`
- Severity has a default value, but is considered to be required
  - The default value is never actually used so the default could be removed, or the severity be required

If this line is becoming too long this is another common TSX syntax I see used, but it isn't used in this codebase

```ts
type Props = {
  children: React.ReactNode;
  severity?: string;
};

export const SmallCallout: React.FC<Props> = ({ children, severity = "warning" }) => {
  return (...);
};
```